### ISSUE TITLE
NativeScript Vue Sample?

### ISSUE THREAD

**Issue by ebellumat** [EddyVerbruggen/nativescript-ar#19]
Any plan to create a demo in NativeScript-Vue version? If not, i can convert?

**Comment by EddyVerbruggen** [comment]
Not really, but it's not too different from [the Angular docs](https://github.com/EddyVerbruggen/nativescript-ar/blob/master/docs/ar-angular.md). I can add a bit of documentation for Vue.

**Comment by ebellumat** [comment]
Im Trying to run this plugin a Vue, but, he asked for the permission, I accept, but the AR Camera does not Start. The log show's me someting, but i cant identify the problem. I Putied the log on pastebin.

- [THE LOG](https://pastebin.com/3V6uQsVA)

- The Vue file.
```vue
<template>
    <Page>
      <ActionBar title="NativeScript AR"></ActionBar>

      <GridLayout class="page">
        <AR v-if="loaded"
          debugLevel="FEATURE_POINTS"
          detectPlanes="true"
          :planeMaterial="planeMaterial">
        </AR>
      </GridLayout>
    </Page>
</template>

<script>
  import { Color } from "tns-core-modules/color";
  export default {
    data() {
      return {
        msg: 'Hello World!',
        loaded: false,
        planeMaterial: {
          diffuse: new Color("white"),
          transparency: 0.2
        }
      }
    },
    mounted () {
      setTimeout(() => {
        this.loaded = true;
      }, 1000);
    }
  }
</script>

<style scoped>
    ActionBar {
        background-color: #53ba82;
        color: #ffffff;
    }

    .title {
        text-align: left;
        padding-left: 16;
    }

    .message {
        vertical-align: center;
        text-align: center;
        font-size: 20;
        color: #333333;
    }

    .drawer-header {
        padding: 50 16 16 16;
        margin-bottom: 16;
        background-color: #53ba82;
        color: #ffffff;
        font-size: 24;
    }

    .drawer-item {
        padding: 8 16;
        color: #333333;
        font-size: 16;
    }
</style>
```

- The main.js file

```js
import Vue from 'nativescript-vue'
import App from './components/App'
// Prints Vue logs when --env.production is *NOT* set while building
Vue.config.silent = (TNS_ENV === 'production')

Vue.registerElement('RadSideDrawer', () => require('nativescript-ui-sidedrawer').RadSideDrawer)
Vue.registerElement("AR", () => require("nativescript-ar").AR);

new Vue({
  render: h => h('frame', [h(App)])
}).$start()
```

**Comment by EddyVerbruggen** [comment]
Try assigning a height and width to the AR view tag. Otherwise, share a repo and I may take a look.

**Comment by ebellumat** [comment]
> Try assigning a height and width to the AR view tag. Otherwise, share a repo and I may take a look.

Oh, Thanks! Here is the repo [Ar-Native](https://github.com/ebellumat/ar-native), and the [Invite](https://github.com/ebellumat/ar-native/invitations)

**Comment by ebellumat** [comment]
This part on the log call me attention.

```CONSOLE ERROR file:///app/vendor.js:4915:20: [Vue warn]: Error in nextTick: "TypeError: undefined is not an object (evaluating'this.nativeView.addSubview')" ```

I'm Creating this sample to present NativeScript on [Vue MeetUp in Vitoria](https://www.meetup.com/pt-BR/Vue-js-in-Vix/events/257782181/), in Brazil, next tuesday

**Comment by ebellumat** [comment]
Following this tutorial for Angular, i got the same error. I'm using iOS 12.1.2.

[Getting Started with Augmented Reality in NativeScript
](https://www.nativescript.org/blog/getting-started-with-augmented-reality-in-nativescript)

`file:///app/tns_modules/nativescript-ar/ar.js:164:24: JS ERROR TypeError: undefined is not an object (evaluating 'this.nativeView.addSubview')`

`*** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: TypeError: undefined is not an object (evaluating 'this.nativeView.addSubview') `

- [THE LOG](https://pastebin.com/HraFCy3U)

**Comment by EddyVerbruggen** [comment]
Hey @ebellumat, I just released 0.6.2 which should fix that issue. It was an incompatibility with NativeScript 5 (4 worked).
